packages/omg-js-util/src/waitForRootchainTransaction.js

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 OmiseGO Pte Ltd
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+async function waitForRootchainTransaction ({
+  web3,
+  transactionHash,
+  millisToWaitForTxn,
```

Comment:
can we rename this parameter, it feels like wait tx for x mill then done. probably intervalCheck or something instead?